src/ssl.c

```diff
@@ -14568,7 +14568,38 @@ int wolfSSL_set_compression(WOLFSSL* ssl)
                    ssl->options.haveStaticECC, ssl->options.haveAnon,
                    ssl->options.side);
     }
+    #ifdef OPENSSL_EXTRA
+    /**
+     * set call back function for psk session use
+     * @param ssl  a pointer to WOLFSSL structure
+     * @param cb   a function pointer to wc_psk_use_session_cb
+     * @return none
+     */
+    void wolfSSL_set_psk_use_session_callback(WOLFSSL* ssl, 
+                                                wc_psk_use_session_cb_func cb)
+    {
+        byte haveRSA = 1;
+        int  keySz   = 0;
+        
+        WOLFSSL_ENTER("wolfSSL_set_psk_use_session_callback");
```

Comment:
Need to add corresponding WOLFSSL_LEAVE() call at end of function.

Reply:
Added WOLFSSL_LEAVE()